mainframer.sh

```diff
@@ -40,32 +42,39 @@ if [ -z "$BUILD_COMMAND" ]; then
 	exit 1
 fi
 
-pushd "$PROJECT_DIR"
+# Read local exclude rules.
+LOCAL_EXCLUDE_FILE=".mainframerignorelocal"
```

Comment:
I'm thinking about separate folder for mainframer configs where we could place .ignorelocal,  .ignoreremote and #50 .config